spec/std/callstack_spec.cr

```diff
@@ -17,9 +17,9 @@ describe "Backtrace" do
     output = `#{tempfile.path}`
 
     # resolved file line:column
-    output.should match(/#{sample} 3:10/) # callee1
-    output.should match(/#{sample} 15:3/) # callee3
-    output.should match(/#{sample} 17:1/) # ???
+    output.should match(/callee1 at #{sample} 3:10/)
+    output.should match(/callee3 at #{sample} 15:3/)
```

Comment:
I rebase this over master locally and I get that class name and type return information is present in OSX.

The following make things pass in osx:
```
    output.should match(/Kls#callee1:Nil at #{sample} 3:10/)
    output.should match(/callee3:Nil at #{sample} 15:3/)
```